src/js/actions/superselect.js

```diff
@@ -182,15 +192,15 @@ define(function (require, exports) {
     };
 
     /**
-     * Removes all artboards from the given ids
-     * We use this to prevent artboards from being cmd+clickable
-     *
+     * Gets all artboards from the given ids
+     * We use this to re-add name badges back into the clickable areas
+     * 
      * @param {LayerStructure} layerTree
      * @param {Immutable.List.<number>} ids
      * @return {Immutable.Iterable.<Layer>}
      */
-    var _removeArtboardIDs = function (layerTree, ids) {
-        return ids.filterNot(function (id) {
+    var _getArtboardIDs = function (layerTree, ids) {
+        return ids.filter(function (id) {
             var layer = layerTree.layers.get(id);
             
             return layer ? layer.isArtboard : true;
```

Comment:
I think this was added recently due to getting null layers during new layer creation / click happening too fast (the newly created layer wouldn't be in our models yet)